### PR TITLE
Add a really smooth and cool transition

### PR DESCRIPTION
**Pull request by AnnieLeonia**

### CODE REVIEW

File: client/src/components/CurrentSpot.vue [hunk 1]
```diff
@@ -1,23 +1,53 @@
...
+    :class="showSpotDetails ? 'large' : showSpotInfo ? 'small' : 'hide'"
+  >
+    <div class="qrspot-container" @click="toggle()">
+      <div class="qrspot--title">{{ currentSpot.title }}</div>
+      <div class="qrspot--info">
+        <div class="qrspot--info--distance">
+          <img alt="My Location" class="distance-icon" :src="distanceIcon" />
+          <span>{{ distanceToMarker() }}</span>
+        </div>
+        <div class="qrspot--info--raiting">
```
Comment (VictorWinberg) [inline client/src/components/CurrentSpot.vue:13]:
```suggestion
      <div class="qrspot-title">{{ currentSpot.title }}</div>
      <div class="qrspot-info">
        <div class="qrspot-info__distance">
          <img alt="My Location" class="distance-icon" :src="distanceIcon" />
          <span>{{ distanceToMarker() }}</span>
        </div>
        <div class="qrspot-info__rating">
```

File: client/src/components/CurrentSpot.vue [hunk 2]
```diff
@@ -26,22 +56,116 @@ export default Vue.extend({
...
+  }
+
+  &.large {
+    right: 0;
+    bottom: 0;
+    left: 0;
+    height: 70%;
+    transition: all 200ms 0ms, height 500ms;
+  }
+
```
Comment (VictorWinberg) [inline client/src/components/CurrentSpot.vue:132]: Un-nest classes (except `.distance-icon` and `.star-icon`)

File: client/src/components/CurrentSpot.vue [hunk 3]
```diff
@@ -26,22 +56,116 @@ export default Vue.extend({
...
+  .qrspot-container {
+    height: 100%;
+    color: $black;
+    cursor: pointer;
+    background-color: $white;
+    border-radius: 2px;
+    box-shadow: $shadow-color;
+
+    .qrspot--title {
```
Comment (VictorWinberg) [inline client/src/components/CurrentSpot.vue:141]:
```suggestion
    .qrspot-title {
```

File: client/src/components/CurrentSpot.vue [hunk 4]
```diff
@@ -26,22 +56,116 @@ export default Vue.extend({
...
+      padding: 10px;
+      font-size: 22px;
+    }
+
+    .qrspot--info {
```
Comment (VictorWinberg) [inline client/src/components/CurrentSpot.vue:146]:
```suggestion
    .qrspot-info {
```

File: client/src/components/CurrentSpot.vue [hunk 5]
```diff
@@ -26,22 +56,116 @@ export default Vue.extend({
...
+      display: flex;
+      justify-content: center;
+      max-width: 500px;
+      margin: 0 auto;
+      font-size: 14px;
+
+      .qrspot--info--distance {
```
Comment (VictorWinberg) [inline client/src/components/CurrentSpot.vue:153]:
```suggestion
      .qrspot-info__distance {
```

File: client/src/components/CurrentSpot.vue [hunk 6]
```diff
@@ -26,22 +56,116 @@ export default Vue.extend({
...
+        padding: 0 10px;
+
+        .distance-icon {
+          height: 14px;
+        }
+      }
+
+      .qrspot--info--raiting {
```
Comment (VictorWinberg) [inline client/src/components/CurrentSpot.vue:161]:
```suggestion
      .qrspot-info__rating {
```

File: client/src/components/GoogleMaps.vue [hunk 7]
```diff
@@ -157,34 +145,25 @@ export default Vue.extend({
...
-        setTimeout(
-          () => this.handleMarkerClick(value, this.markerSelected),
-          200
-        );
-        this.markerSelected = -1;
-        this.showSpotDetails = true;
-      }
+      // const informationBoxDiv = document.getElementById("geocache-info");
+      // const { BOTTOM } = google.maps.ControlPosition;
+      // this.map.controls[BOTTOM].push(informationBoxDiv);
```
Comment (VictorWinberg) [inline client/src/components/GoogleMaps.vue:160]: Anything to save?

**Review by VictorWinberg — CHANGES_REQUESTED** [review]

File: client/src/components/GoogleMaps.vue (hunk 7 above)
Comment (AnnieLeonia) [reply client/src/components/GoogleMaps.vue]: Not really, I have removed it!

File: client/src/components/CurrentSpot.vue [hunk 8]
```diff
@@ -1,23 +1,53 @@
...
+    :class="showSpotDetails ? 'large' : showSpotInfo ? 'small' : 'hide'"
+  >
+    <div class="qrspot-container" @click="toggle()">
+      <div class="qrspot-title">{{ currentSpot.title }}</div>
+      <div class="qrspot-info">
+        <div class="qrspot-info__distance">
+          <img alt="My Location" class="distance-icon" :src="distanceIcon" />
+          <span>{{ distanceToMarker() }}</span>
+        </div>
+        <div class="qrspot-info__raiting">
```
Comment (VictorWinberg) [inline client/src/components/CurrentSpot.vue:13]:
```suggestion
        <div class="qrspot-info__rating">
```

File: client/src/components/CurrentSpot.vue [hunk 9]
```diff
@@ -26,22 +56,116 @@ export default Vue.extend({
...
+
+.qrspot-info__distance {
+  padding: 0 10px;
+
+  .distance-icon {
+    height: 14px;
+  }
+}
+
+.qrspot-info__raiting {
```
Comment (VictorWinberg) [inline client/src/components/CurrentSpot.vue:164]:
```suggestion
.qrspot-info__rating {
```

**Review by VictorWinberg — APPROVED** [review]
LGTM 😎

**Pull request merged**